app/Models/Characters/Monster.php

```diff
@@ -31,6 +32,38 @@ public function skills()
         return $this->belongsToMany(Skill::class)->withPivot('bonus');
     }
 
+    public function items()
+    {
+        return $this->belongsToMany(Item::class);
```

Comment:
Makes me happy when we can write a relation like this instead of ->hasMany(Item::class, 'id', 'oThEr_iD', 'something_else', 'ohgodwhatevenisthis')